static/js/apps/explore/app.tsx

```diff
@@ -124,44 +126,50 @@ export function App(props: { isDemo: boolean }): JSX.Element {
     ? null
     : savedContext.current[0]["insightCtx"];
   return (
-    <Container className="explore-container">
-      {props.isDemo && (
-        <AutoPlay
-          autoPlayQuery={autoPlayQuery}
-          inputQuery={setQuery}
-          disableDelay={loadingStatus === LoadingStatus.DEMO_INIT}
-        />
-      )}
-      {loadingStatus === LoadingStatus.FAILED && (
-        <ErrorResult
-          query={query}
-          debugData={debugData}
-          exploreContext={exploreContext}
-          queryResult={queryResult}
-          userMessage={userMessage}
-        />
-      )}
-      {loadingStatus === LoadingStatus.LOADING && (
-        <div>
-          <Spinner isOpen={true} />
-        </div>
-      )}
-      {loadingStatus === LoadingStatus.DEMO_INIT && (
-        <div className="row explore-charts">
-          <SearchSection query={query} debugData={null} exploreContext={null} />
-        </div>
-      )}
-      {loadingStatus === LoadingStatus.SUCCESS && (
-        <SuccessResult
-          query={query}
-          debugData={debugData}
-          exploreContext={exploreContext}
-          queryResult={queryResult}
-          pageMetadata={pageMetadata}
-          userMessage={userMessage}
-        />
-      )}
-    </Container>
+    <RawIntlProvider value={intl}>
```

Comment:
mention in the PR description that you're also adding i18n support in this PR

Reply:
done